.github/workflows/integration-tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Checkout the repo
         uses: actions/checkout@f25a3a9f25bd5f4c5d77189cab02ff357b5aedeb # v2.4.1
       - name: Run Tests
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@010068ddc7dd55cd4439159a485390aa651fdd39 # v2.0.3
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@b919a5a6ed9bba641158ffbb636bcafdfb60031e # v2.0.3
```

Comment:
Did you make a new release on the github-actions repo. I don't think this is technically 2.0.3 any more

Reply:
Fixed